es/18_3_Accediendo_a_Bitcoind_con_NodeJS.md

```diff
@@ -0,0 +1,242 @@
+$ cd bcrpc
+$ npm install
+```
+
+Para probar el paquete BCRPC, primero debe establecer variables ambientales para rpcuser y rpcpassword. Como de costumbre, estas provienen de `~/. bitcoin/bitcoin.conf`. También debe establecer el puerto RPC en 18332 que debería ser correcto para la configuración estándar de testnet descrita en estos documentos.
+
+```
+$ export BITCOIND_USER=StandUp
+$ export BITCOIND_PASS=d8340efbcd34e312044c8431c59c792c
+$ export BITCOIND_PORT=18332
+```
+> :warning: **ADVERTENCIA:** Obviamente, nunca debería poner su contraseña en una variable ambiental en un entorno de producción.
+
+> :link: **MAINNET VS TESTNET:** El puerto sería 8332 para una configuración de mainnet.
+
+Ahora puede verificar que todo funciona correctamente:
+```
+$ npm test
+
+> bcrpc@0.2.2 test /home/user1/bcrpc
+> mocha tests.js
+
+  BitcoinD
+    ✓ is running
+
+  bcrpc
+    ✓ can get info
+
+  2 passing (36ms)
+```
+Felicitaciones, ahora tiene un envoltorio RPC listo para Bitcoin para Node.js que está trabajando con su configuración de Bitcoin.
+
+### Crear un proyecto BCRPC
+
+Ahora puede crear un nuevo proyecto Node.js e instalar BCRPC a través de npm.
+```
+$ cd ~
+$ mkdir myproject
+$ cd myproject
+$ npm init
+  [continue with default options]
+$ npm install bcrpc
+```
+# Construya su conexión
+
+En su directorio `myproject` cree un archivo `.js` donde se ejecutará el código JavaScript. 
+
+Puede iniciar una conexión RPC creando un `RpcAgent`:
+```
+const RpcAgent = require('bcrpc');
+agent = new RpcAgent({port: 18332, user: 'StandUp', pass: 'd8340efbcd34e312044c8431c59c792c'});
+```
+Obviamente, su usuario `` y `pass` deberían coincidir de nuevo con lo que está en su `~/. bitcoin/bitcoin.conf`, y usar `port 18332` si está en testnet.
+
+### Hacer una llamada RPC
+
+Usando BCRPC, puede usar los mismos comandos RPC que usaría normalmente a través de `bitcoin-cli` con su `RpcAgent`, excepto que necesitan estar en camelCase. Por ejemplo, `getblockhash` sería `getBlockHash`.
+
+Para imprimir el nuevo número de bloque, sólo tiene que llamar a `getBlockCount` a través de su `RpcAgent`:
+```
+  agent.getBlockHash(blockCount.result, function (err, hash) {
+    if (err)
+      throw Error(JSON.stringify(err));
+    console.log(hash.result);
+  })
+```
+El resultado de las funciones BCRPC es un objeto JSON que contiene información sobre cualquier error y el id de la solicitud. Al acceder a su resultado, se añade `.result` al final de la misma para especificar que usted está interesado en el resultado real, no información sobre errores. 
+
+### Ejecute su código
+
+Puede encontrar el código `getinfo` en el [directorio src]((src/17_3_getinfo.js).
+```
+$ node getinfo.js
+1831094
+00000000000002bf8b522a830180ad3a93b8eed33121f54b3842d8838580a53c
+```
+Esto es lo que la salida del ejemplo anterior se vería si se reemplaza `consola.log(blockCount.result);` `consola.log(hash.result);` consola.log(blockCount);` y `consola.log(hash);`, respectivamente:
```

Comment:
missing \` right before consola.log(blockCount)

Reply:
Additionally, console.log is javascript syntax and should be left untranslated like other code blocks. They should all be console.log() instead of consola.log()